src/app/features/search/search.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="govuk-grid-row search-page">
+<div class="govuk-grid-row search-page" *appUserRole="['CVSFullAccess', 'TechRecord.View']">
```

Comment:
Same as before. rather than letting users in here and hiding content, would it not be better to guard the route?